torch/testing/_internal/common_utils.py

```diff
@@ -956,14 +996,14 @@ def assertNotEqual(self, x, y, prec=None, message=''):
                 # Use `item()` to work around:
                 # https://github.com/pytorch/pytorch/issues/22301
                 max_err = diff.max().item()
-                self.assertGreaterEqual(max_err, prec, message)
+                self.assertGreater(max_err, atol, message)
```

Comment:
Any particular reason for the change here? Seems subtle.

Reply:
assertGreaterEqual won't work when using a tolerance of zero (exact match).
e.g. in `assertNotEqual( 5, 5, atol=0 )`, if you're doing `assertGreaterEqual(5-5, 0)` you're going to pass when you should fail.